packages/messenger-widget/src/components/Menu/Menu.tsx

```diff
@@ -0,0 +1,59 @@
+import './Menu.css';
+import { useContext } from 'react';
+import closeIcon from '../../assets/images/cross.svg';
+import addIcon from '../../assets/images/add.svg';
+import settingsIcon from '../../assets/images/settings.svg';
+import { LeftViewSelected, UiViewStateType } from '../../utils/enum-type-utils';
+import { GlobalContext } from '../../utils/context-utils';
+import { startLoader } from '../Loader/Loader';
+
+export default function Menu() {
+    // fetches context api data
+    const { dispatch } = useContext(GlobalContext);
+
+    const showContactList = () => {
+        dispatch({
+            type: UiViewStateType.SetSelectedLeftView,
+            payload: LeftViewSelected.Contacts,
+        });
+        startLoader();
+    };
+
+    return (
+        <div className="menu-container height-fill width-fill">
+            <div className="menu-item-cancel d-flex justify-content-end">
+                <img
+                    src={closeIcon}
+                    alt="close"
+                    className="pointer-cursor close-icon"
+                    onClick={() => showContactList()}
+                />
+            </div>
+            {/*  eslint-disable */}
+            <div className="d-flex align-items-center justify-content-start pointer-cursor menu-items font-weight-400 text-primary-color">
```

Comment:
Did you disable eslint because of the line length?
There should be a git hook running `prettier --write` and `eslint --fix` for all staged files